hazelcast/serialization/compact.py

```diff
@@ -1754,13 +1760,14 @@ def _init(self):
 
         for field in self.fields_list:
             kind = field.kind
-            if FIELD_OPERATIONS[kind].is_var_sized():
+            if kind < 0 or kind >= FieldKind.NOT_AVAILABLE:
+                raise HazelcastSerializationError(f"Invalid field kind: {kind}")
```

Comment:
Do we need this check? There is no way FieldKind.NOT_AVAILABLE can be added to a Schema fields. Or there can never be negative FieldKind because there is not any FIeldKind like that.